src/server/api/controllers/users.controller.js

```diff
@@ -1,16 +1,41 @@
 /* TODO: This is an example controller to illustrate a server side controller.
 Can be deleted as soon as the first real controller is added. */
 
+//const moment = require('moment-timezone');
+
 const knex = require('../../config/db');
-const moment = require('moment-timezone');
 
 const createUser = async (body) => {
+  const newUser = {
+    user_name: body.userName,
+    email: body.email,
+    profile_image: body.profileImage,
+    last_seen: new Date(),
+  };
+  await knex('users').insert(newUser);
+  return {
```

Comment:
No need to return these properties. Especially since you sent them from the UI. You only need to return the id that is generated.